jupyter/datascience/ubi9-python-3.11/setup-elyra.sh

```diff
@@ -14,6 +14,11 @@ if [ "$(ls -A /opt/app-root/runtimes/)" ]; then
   cp -r /opt/app-root/runtimes/..data/*.json $(jupyter --data-dir)/metadata/runtimes/
 fi
 
+# Set elyra runtime images json from volume mount
+if [ "$(ls -A /opt/app-root/pipeline-runtimes/)" ]; then
```

Comment:
Probably a nitpick, but wouldn't be a bit safer to check the full directory path from where we are copying the json files?
```
```suggestion
if [ "$(ls -A /opt/app-root/pipeline-runtimes/..data/)" ]; then
```
```

Reply:
Thanks for the suggestion! I see the reasoning behind checking ..data/ directly, but in the case of a Kubernetes-mounted ConfigMap or Secret, /opt/app-root/pipeline-runtimes/ is already the correct path to check.

Since Kubernetes manages this directory and automatically updates symlinks to point to the latest version in ..data/, checking ..data/ explicitly is not necessary and could introduce unnecessary dependencies on Kubernetes' internal structure.

Reply:
Well, we're depending on that internal structure already on the very next line in this code anyway :slightly_smiling_face: 

Also, I now realized that this check will always be true won't it be? Since we're creating this directory during the [image build here](https://github.com/opendatahub-io/notebooks/pull/909/files#diff-bf1b39ab6e296ed2806e2d5b1c26436ded10fb59f70b1e94334942808596193dR94). So if I'm not missing anything, the check in its current state doesn't make much sense?

Reply:
hmm

```
❯ mkdir empty
❯ [ $(ls -A empty) ] || echo false
false


❯ touch empty/file
❯ [ $(ls -A empty) ] || echo false
```

nothing printed in second case

Reply:
Okay, I see, I didn't read the condition properly. So the condition works as expected this way. Sorry about that.